File: python/src/bedrock_bio/list_namespaces.py
from .config import config


def list_namespaces() -> list[str]:
    """List available namespaces (data sources).

    Returns
    -------
    list[str]
        Namespace identifiers.

    Examples
    --------
    >>> import bedrock_bio as bb
    >>> bb.list_namespaces()
    ['ukb_ppp', ...]
    """
    return list(config.get_namespaces().keys())

File: r/R/list_namespaces.R
#' List available namespaces (data sources)
#'
#' @returns A character vector of namespace identifiers.
#'
#' @examples
#' \dontrun{
#' library(bedrockbio)
#' list_namespaces()
#' }
#'
#' @export
list_namespaces <- function() {
  names(get_namespaces())
}
